### ISSUE TITLE
[Accessibility]: Insufficient contrast of `danger` color used for errors on dark theme

### ISSUE THREAD

**Issue by kopancek** [sourcegraph/sourcegraph#35700]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34111

### Problem description

There is insufficient contrast for error texts when using a dark theme. We are using a CSS variable for the red color - `--danger` which does not have good contrast against dark background. We should use a different red color for dark theme to improve contrast ratio.

### Expected behavior

Contrast ratio of error text adheres to the spec even on dark theme

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by felixfbecker** [comment]
@kopancek where exactly does the red danger text occur in the UI and against what background?
